pennylane/operation.py

```diff
@@ -2000,7 +2001,9 @@ def sparse_matrix(self, wires=None):  # pylint:disable=arguments-renamed
             idx = wires.index(o.wires)
             list_of_sparse_ops[idx] = coo_matrix(o.matrix())
 
-        return functools.reduce(lambda i, j: kron(i, j, format="coo"), list_of_sparse_ops)
+        return functools.reduce(lambda i, j: kron(i, j, format="coo"), list_of_sparse_ops).asformat(
```

Comment:
Can it be `format="csr"`?

Reply:
In my tests, I found a very similar time between the two approaches.
I'm happy with changing this, though.